package.json

```diff
@@ -38,11 +38,17 @@
         "onCommand:leetcode.testSolution",
         "onCommand:leetcode.submitSolution",
         "onCommand:leetcode.switchDefaultLanguage",
+        "onCommand:leetcode.signinByCookie",
         "onView:leetCodeExplorer"
     ],
     "main": "./out/src/extension",
     "contributes": {
         "commands": [
+            {
+                "command": "leetcode.signinByCookie",
+                "title": "Sign In By Cookie",
```

Comment:
nit: `By` should be lower case since it's a preposition

Reply:
Ok, I will change it right now

Reply:
> "command": "leetcode.signinByCookie", ? 
This "by" also?